tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
```

Comment:
@kaushik94 I just checked, and I think it is necessary because the maximum line length is supposed to be 79 characters I think?